dive_core/common/gpudefs.h

```diff
@@ -80,4 +81,143 @@ inline uint16_t DecodeSGPRs(uint32_t pgm_rsrc1)
     return sgprs;
 }
 
+//--------------------------------------------------------------------------------------------------
```

Comment:
is there a reason you prefer to put this in gpudefs.h? is it better to put it in trace_stats.h?